tests/platform_tests/test_cont_warm_reboot.py

```diff
@@ -116,7 +120,9 @@ def check_neighbors(dut):
           "BGP neighbor's ASN does not match minigraph")
 
 
-def test_cont_warm_reboot(duthost, localhost, conn_graph_facts, continuous_reboot_count, continuous_reboot_delay):
+@pytest.mark.usefixtures('get_advanced_reboot')
+def test_cont_warm_reboot(duthost, ptfhost, localhost, conn_graph_facts, continuous_reboot_count, \
+    continuous_reboot_delay, enable_continuous_io, get_advanced_reboot):
```

Comment:
The `enable_continuous_io` fixture is declared as autouse. It's unnecessary to add it here in the argument list here.
Or you can remove the `autouse=True` for `enable_continuous_io` and explicitly specify it here in argument list. Personally I prefer the later approach for this case because 'Explicit is better than implicit`.

Reply:
Thanks for the explanation here. I moved the fixtures to this script and set them to autouse for the module. It is explicit in the script now, but implicit in the function. Does it look good?

Reply:
I think you can remove all the autouse fixtures from the argument list. 

* continuous_reboot_count
* continuous_reboot_delay
* enable_continuous_io

They will be called automatically.

Reply:
I think I still have to keep them in the argument list. The benefit of setting them `autouse=Tue` is that I don't have to call decorator `@pytest.mark.usefixtures()` for these arguments.
For example, after removing `continuous_reboot_count` fixture, below error was thrown:
```
        # Start continuous warm reboot on the DUT
>       for _ in range(continuous_reboot_count):
E       TypeError: range() integer end argument expected, got function.
```

Reply:
Ah, my understanding was wrong. Sorry, I apologize! 
If a fixture is marked as autouse, it is automatically called when it is in scope. No need to add it to the argument list of test case function. But if the test case need to use some value returned from the fixture, the fixture must be explicitly declared in its argument list.

Reply:
That is all good. I am happy that I could learn a few things from these experiments.